pkg/driver/driver.go

```diff
@@ -203,31 +202,3 @@ func (p *OpenstackDriver) GetVolumeIDs(_ context.Context, req *driver.GetVolumeI
 	}
 	return &driver.GetVolumeIDsResponse{VolumeIDs: names}, nil
 }
-
-// GenerateMachineClassForMigration helps in migration of one kind of machineClass CR to another kind.
```

Comment:
we can also remove the kind `openStackMachineClassKind`

Reply:
Already done, PTAL :)